### PR TITLE
Fixed challenge description

### LINKED ISSUE #11092: Clarify what the `.pop()` method does
Maybe you should add "The pop() method removes the last element from an array and returns that element".

Challenge [Manipulate Arrays With pop](https://www.freecodecamp.com/challenges/manipulate-arrays-with-pop) has an issue.
User Agent is: <code>Mozilla/5.0 (iPad; CPU OS 10_0_2 like Mac OS X) AppleWebKit/602.1.50 (KHTML, like Gecko) Version/10.0 Mobile/14A456 Safari/602.1</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript

// Example
var ourArray = [1,2,3];
var removedFromOurArray = ourArray.pop(); 
// removedFromOurArray now equals 3, and ourArray now equals [1,2]

// Setup
var myArray = [["John", 23], ["cat", 2]];

// Only change code below this line.
var removedFromMyArray = myArray.pop();



```

### PR DESCRIPTION
**Pull request by hristijankiko**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX
#### Description

closes #11092

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/basic-javascript.json [hunk 1]
```diff
@@ -2048,9 +2048,8 @@
       "title": "Manipulate Arrays With pop()",
       "description": [
         "Another way to change the data in an array is with the <code>.pop()</code> function.",
-        "<code>.pop()</code> is used to \"pop\" a value off of the end of an array. We can store this \"popped off\" value by assigning it to a variable.",
+        "<code>.pop()</code> is used to \"pop\" a value off of the end of an array. We can store this \"popped off\" value by assigning it to a variable. In other words, <code>.pop()</code> removes the last element from an array and returns that element.",
         "Any type of entry can be \"popped\" off of an array - numbers, strings, even nested arrays.",
-        "For example, for the code<br><code>var oneDown = [1, 4, 6].pop();</code><br>the variable <code>oneDown</code> now holds the value <code>6</code> and the array becomes <code>[1, 4]</code>.",
```
Comment (atjonathan) [inline seed/challenges/01-front-end-development-certification/basic-javascript.json]: According to [@erictleung's comment](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/11092#issuecomment-252470111), should there not be a code snippet here?

Comment (hristijankiko) [inline seed/challenges/01-front-end-development-certification/basic-javascript.json]: Oops, looks like i missed it, I'm going to add it in now.

**Comment by hristijankiko** [comment]
I think it should be good now.

**Review by ghost — APPROVED** [review]
#### LGTM :+1:

Thanks for contributing to the most awesome open source community and happy hacktoberfest!

**Pull request merged**